analyzeme/src/file_formats/v7.rs

```diff
@@ -0,0 +1,81 @@
+//! This module implements file loading for the v7 file format used until
+//! crate version 9.2.0
+
+use std::{error::Error, path::Path};
+
+use analyzeme_9_2_0::ProfilingData;
+use decodeme::{
+    event::Event,
+    event_payload::{EventPayload, Timestamp},
+    lightweight_event::LightweightEvent,
+    Metadata,
+};
+
+pub const FILE_FORMAT: u32 = analyzeme_9_2_0::CURRENT_FILE_FORMAT_VERSION;
+
+#[derive(Debug)]
+pub struct EventDecoder {
+    legacy_profiling_data: ProfilingData,
```

Comment:
nit: not sure that the `legacy_` prefix is needed.

Reply:
I'd like to keep that one. It's a bit verbose but I'd rather be too blunt than too subtle here.